packages/gestalt/src/Badge.css

```diff
@@ -1,16 +1,14 @@
 .Badge {
   composes: antialiased sansSerif from "./Typography.css";
-  composes: fontWeightBold from "./Typography.css";
-  composes: blue from "./Colors.css";
   composes: inlineBlock from "./Layout.css";
-  border-radius: 4px;
-  border-style: solid;
-  border-width: 1px;
+  composes: rounding1 from "./Borders.css";
+  composes: white from "./Colors.css";
+  composes: fontWeightBold from './Typography.css';
   font-size: 10px;
-  line-height: 1;
   margin-top: -4px;
+  min-height: 12px;
```

Comment:
@denamwangi Do we need `min-height`? The `text` prop is required so it should already have `font-size` (10px) + padding (2px top/bottom)

Reply:
you're so right- thanks